src/order.js

```diff
@@ -0,0 +1,9 @@
+const cart = [{
+    code: 'Bath Bomb Pink',
+    quantity: 2,
+}, {
+    code: 'Bath Bomb Orange',
+    quantity: 6,
+}];
+
+export default cart;
```

Comment:
re: naming - I'd stay consistent with file names. If this "order.js" file only contains one variable (cart), I'd name the file "cart.js"